code/game/objects/items/storage/boxes.dm

```diff
@@ -74,6 +74,45 @@
 		return 0
 	return ..()
 
+//Locker overloading issue solving boxes
+/obj/item/storage/box/suitbox
+	name = "compression box of invisible outfits"
+	desc = "a box with bluespace compression technology that nanotrasen has approved, but this is extremely heavy... If you're glued with this box, pull out of the contents and fold the box."
+	w_class = WEIGHT_CLASS_HUGE
+	drag_slowdown = 4 // do not steal by dragging
+	/* Note for the compression box:
+		Do not put any box (or suit) into this box, or it will allow infinite storage.
+		non-storage items are only legit for this box. (suits are storage too, so, no.)
```

Comment:
add a test for this in code so nobody does it

Reply:
it's obvious text enough, I think

Reply:
you seriously overestimate coder's ability to see things